src/main/scad/laszlo-molnar/no-holes-barred/molnar.no-holes-barred.scad

```diff
@@ -0,0 +1,58 @@
+include <puzzlecad.scad>
+
+box_puzzle_border = 6;
+box_puzzle_top_inset = 1;
+
+$burr_inset = 0.12;
+$burr_bevel = 1;
+$unit_beveled = true;
+$burr_scale = 16;
+
+dim = $burr_scale * 3 + box_puzzle_border * 2 + $burr_inset * 2;
+hight = $burr_scale * 3 + box_puzzle_border + box_puzzle_top_inset + $burr_inset;
+
+box();
+*piece_x4();
+*piece_x1();
```

Comment:
I suggest we combine these into a single burr_plate pieces() with five pieces on it. (Ok to keep the piece_triangle() separate)